backend/tests/utils.js

```diff
@@ -21,10 +21,10 @@ const createCandidates = async (numOfMatches, elos) => {
   for (let i = 0; i < numOfMatches.length; i++) {
     query.push({
       _id: candidateIds(i),
-      name: '',
+      name: 'a',

```

Comment:
Instead of just 'a' why don't we make this a little more intuitive and make it 'candidate' + index (e.g. 'candidate 5'). Even though the field isn't used for testing right now it can help if we want to make a more robust test for fields down the road and is more straightforward.

Reply:
+1, maybe same for major below as well